### ISSUE TITLE
[SORC] Waagh/Chop Fasta - Visual cooldown on icon of Infernal Wave ability

### ISSUE THREAD

**Issue by GonTaa** [WarEmu/WarBugs#18268]
Hello.

First of all, a huge thanks to the dev for their work, which makes playing on this server possible.

Second, this is **NOT** a post about (cdr) Waaagh or Chop Fasta not affecting the cooldown of Infernal Wave when you already have the tactic Neverending Agony. It's about the visual effects that happens on the icon of Infernal Wave.

**Expected behavior and actual behavior:**

Expected behavior : right as I cast the ability Infernal Wave while under Chop Fasta or Waagh, the icon of the ability Infernal Wave should displays a 5 second timer.

Actual behavior : right as I cast Infernal Wave while under Chop Fasta or Waagh, the icon of the ability Infernal Wave displays no cooldown, as if the ability was available, but it's not. 

Therefore, you can't know for sure if your ability is ready, unless counting 5 seconds in your head , and might waste some precious time clicking on it for no result.

**Steps to reproduce the problem:**

1) Take a sorc
2) Slot the tactic Neverending Agony
3) Group a Black Orc or a Choppa
4) Use Waagh/Chop Fasta
5) Use Infernal Wave
==> Ability icon of Infernal Wave is visually messed up

**Comment by GonTaa** [comment]
Hello dear devs ! 

Any news on this issue ? Just out of curiosity :) 

Again, thanks a lot for all your hard work !

**Comment by Niall86** [comment]
So Waagh/Chop Fasta and Neverending Agony not stacking is a supposed behavior?

**Comment by GonTaa** [comment]
Yes. It used to work on Live but on RoR it doesnt work and that's intended.
